process-engine/bpmn-studio
# ✨ Add Remote Solution History

## Changes

1. Add Remote Solution History

PR: #1538

## How to test the changes

- Add some remote solutions
- **Notice that there will be a remote solution history, as soon as the first remote solution was opened**
- Click on any entry of the remote solution history
- **Notice that the uri remote solution will be written in the input automatically**
- **Notice that there is an online status of the process engine**
